src/AuthModule/Components/ResetPassword/ResetPassword.tsx

```diff
@@ -1,7 +1,172 @@
+    } catch (error) {
+      console.log(error)
+      toast.error(error?.response?.data?.message);
+      setIsLoading(false);
+    }
+    }
+// ########################## show password ##################################
+
+    const[showPassword,setShowPassword]=useState(true);
+const togglePassword=()=>{
+  setShowPassword(!showPassword);
+}
+
+
   return (
-    <div>ResetPassword</div>
+    <>  
+<div className="auth-container vh-100  d-flex flex-column justify-content-center align-items-center">
+        <div className="logo">
+          <img className="form-logo pb-2 " src={logo} alt="logo" />
+        </div>
+        <form
+          onSubmit={handleSubmit(submitNewPassword)}
+          className="form  col-xl-5 col-lg-6 col-md-7 col-sm-9 col-10 bg-info rounded-4 p-sm-5 p-3"
+        >
+          <p className="text-light mb-0 mt-1">welcome to PMS</p>
+          <h2 className="form-Name ">
+            <span>R</span>eset Password
+          </h2>
+
+          <div className="  d-flex flex-column  mt-2">
+
+          <div className=" one-input-group">
+              <div
+                className="form-outline my-1  text-start   "
+                data-mdb-input-init
+              >
+                <label className="form-label  fw-medium mb-0">E-mail</label>
+                <input
+                  {...register("email", emailValidation)}
+                  className="form-input form-control bg-transparent border-0 rounded-bottom-0  border border-bottom text-white p-1"
+                  type="text"
+                  placeholder="Enter your E-mail"
+                  aria-label="readonly input example"
+                />
+                {errors?.email && (
+                  <p className="mt-1  text-danger">{errors?.email?.message}</p>
+                )}
+              </div>
+            </div>
+
+            <div className=" one-input-group">
+              <div
+                className="form-outline my-1  text-start   "
+                data-mdb-input-init
+              >
+                <label className="form-label  fw-medium mb-0">OTP</label>
+                <input
+                  {...register("seed", OTPValidation)}
+                  className="form-input form-control bg-transparent border-0 rounded-bottom-0  border border-bottom text-white p-1"
+                  type="text"
+                  placeholder="Enter your OTP"
+                  aria-label="readonly input example"
+                />
+                {errors?.seed && (
+                  <p className="mt-1  text-danger">{errors?.seed?.message}</p>
+                )}
+              </div>
+            </div>
+
+            <div className=" one-input-group">
+              <div className="form-outline position-relative text-start d-flex flex-wrap">
+                <label className="w-100 form-label fw-medium mb-0">
+                  Password
+                </label>
+                <input
+                  {...register("password", passwordValidation)}
```

Comment:
We might add a validation check for checking that both passwords match, this will ensure that no network request is sent to the backend when the password don't match.